specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2025-11-01-preview/cassandrarbac.json

```diff
@@ -0,0 +1,617 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB",
+    "description": "Azure Cosmos DB Role Based Access Control Resource Provider REST API",
+    "version": "2025-11-01-preview"
```

Comment:
>11 [](http://example.com/codeflow?start=21&length=2)

i assume this is work in progress? no net new changes from previous API version, I am moving it out of ARM feedback queue, please feel free to put it back when ready.